CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 ### Fixes
 
 - Detect AppHangsV2 when tracing not enabled (#5184)
+- Add shared default values for session replay options initializers (#5210)
```

Comment:
`m`: Can we point out which default values we fixed, please?
```suggestion
- Fix default values for session replay options (#5210) TODO add the fixed ones.
```

Reply:
I adapted it, it's only the `enableViewRendererV2` which changed in the initializer.